rts/lra/test/basic/src/test/java/io/narayana/lra/arquillian/ParticipantDataIT.java

```diff
@@ -38,6 +40,8 @@
 public class ParticipantDataIT extends TestBase {
     private static final Logger log = Logger.getLogger(ParticipantDataIT.class);
 
+    private static final String MY_CUSTOM_DATA = "lra-participant-custom-data";
```

Comment:
Thanks Mike for your review. You are right, CUSTOM_DATA sounds better. BTW I deleted that line because I wanted to use the START_DATA and END_DATA constants for the NARAYANA_LRA_PARTICIPANT_DATA_HEADER_NAME header.